tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -847,19 +847,7 @@ public function theAdministratorTriesToAddUserToGroupUsingTheGraphAPI(string $us
 	 * @throws GuzzleException | Exception
 	 */
 	public function theAdministratorTriesToAddUserToNonExistentGroupUsingTheGraphAPI(string $user, ?string $byUser = null): void {
-		$credentials = $this->getAdminOrUserCredentials($byUser);
-		$groupId = WebDavHelper::generateUUIDv4();
-		$userId = $this->featureContext->getAttributeOfCreatedUser($user, "id");
-		$this->featureContext->setResponse(
-			GraphHelper::addUserToGroup(
-				$this->featureContext->getBaseUrl(),
-				$this->featureContext->getStepLineRef(),
-				$credentials['username'],
-				$credentials['password'],
-				$userId,
-				$groupId
-			)
-		);
+		$this->featureContext->setResponse($this->addUserToGroup('', $user, $byUser));
```

Comment:
Would this solve what you're trying to achieve here?
```suggestion
		$this->featureContext->setResponse($this->addUserToGroup('', $user, $byUser,true));
```